src/framework/components/animation/component.js

```diff
@@ -49,7 +49,7 @@ class AnimationComponent extends Component {
      * @param {number} [blendTime] - The time in seconds to blend from the current
      * animation state to the start of the animation being set.
      */
-    play(name, blendTime) {
+    play(name, blendTime = 0) {

```

Comment:
Change JSDocs to say what the default value should be

Reply:
Done.